src/Statistics/WilsonBinomialConfidenceIntervalEstimatorTests.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
+#include "Statistics/BinomialConfidenceInterval.h"
+
+namespace orbit_statistics {
+
+const WilsonBinomialConfidenceIntervalEstimator kEstimator;
```

Comment:
I guess usually you would want to have a clear initialization of each class being used in a test, right be fore the test.
In that case, you would consider adding a "test fixture" and implement a `SetUp` method (or a default constructor).

Alternatively, you would create the class each time explicitly in the test case.

I guess in that case, it is fine, as the Estimator does not contain any state. But it is worth considering.

Reply:
I thought about setting up a fixture, yet it seems like too much of a hassle for a stateless class. So I'll just instantiate it for every test case.